ydb/core/kqp/provider/yql_kikimr_datasource.cpp

```diff
@@ -768,7 +772,15 @@ class TKikimrDataSource : public TDataProviderBase {
                     .Repeat(TExprStep::RewriteIO);
 
                 const auto& query = tableDesc.Metadata->ViewPersistedData.QueryText;
-                return RewriteReadFromView(node, ctx, query, cluster);
+                NKqp::TKqpTranslationSettingsBuilder settingsBuilder(
+                    SessionCtx->Query().Type,
+                    SessionCtx->Config()._KqpYqlSyntaxVersion.Get().GetRef(),
+                    cluster,
+                    query,
+                    SessionCtx->Config().BindingsMode,
+                    GUCSettings
```

Comment:
[GUCSettings](https://pgpedia.info/g/guc.html) are needed here. That is why we pass them in CreateKikimrDataSource.

We cannot, unfortunately,  pass the KQP translation settings builder itself from the KQP host to the Kikimr DataSource, because it needs `query` to be built.